### PR TITLE
Udp len invalid/60x/v2

### PR DESCRIPTION
**Pull request by inashivb**
- https://github.com/OISF/suricata/pull/8472
- https://github.com/OISF/suricata/pull/8464

suricata-verify-pr: 1078

EDIT: suricata verify PR isn't able to check the event behavior so far. it only verifies that the issue does not exist with TCP.

### CODE REVIEW

**Comment by lukashino** [comment]
Looking good from my side :+1:

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 12065

File: rules/decoder-events.rules [hunk 1]
```diff
@@ -68,6 +68,7 @@ alert pkthdr any any -> any any (msg:"SURICATA TCP duplicated option"; decode-ev
 alert pkthdr any any -> any any (msg:"SURICATA UDP packet too small"; decode-event:udp.pkt_too_small; classtype:protocol-command-decode; sid:2200038; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA UDP header length too small"; decode-event:udp.hlen_too_small; classtype:protocol-command-decode; sid:2200039; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA UDP invalid header length"; decode-event:udp.hlen_invalid; classtype:protocol-command-decode; sid:2200040; rev:2;)
+alert pkthdr any any -> any any (msg:"SURICATA UDP invalid length field in the header"; decode-event:udp.len_invalid; classtype:protocol-command-decode; sid:2200220; rev:2;)
```
Comment (victorjulien) [inline rules/decoder-events.rules:71]: should be 2200120 (in master too) and at the bottom next sid should be updated

**Comment by victorjulien** [comment]
SV tests need review for min-version

**Comment by victorjulien** [comment]
If a PR has issues, please set it to draft.

File: rules/decoder-events.rules (hunk 1 above)
Comment (lukashino) [reply rules/decoder-events.rules:71]: How the sid should be derived? Why should it be 2200120?

Comment (victorjulien) [reply rules/decoder-events.rules:71]: It should just be the next sid in that file. Each new sid should just be the last +1

**Pull request closed without merging**